onnxruntime/test/testdata/transform/transform_nested_ifs_toplogical_sorted_nodes.txt

```diff
@@ -0,0 +1,859 @@
+    }
+    domain: ""
+  }
+  node {
+    input: "int64_0_25"
+    input: "self_rank"
+    output: "int64_0_25_cast"
+    name: "n12"
+    op_type: "CastLike"
+    domain: ""
+  }
+  node {
+    input: "self_rank"
+    input: "int64_0_25_cast"
+    output: "cond_26"
+    name: "n13"
+    op_type: "Equal"
+    domain: ""
+  }
+  node {
+    input: "cond_26"
+    output: "result_29"
+    name: "n14"
+    op_type: "If"
+    attribute {
+      name: "then_branch"
+      g {
+        node {
+          input: "result_24"
+          output: "result_27"
+          name: "n0"
+          op_type: "Squeeze"
+          domain: ""
+        }
+        name: "thenGraph_27"
+        output {
+          name: "result_27"
+          type {
+          }
+        }
+      }
+      type: GRAPH
+    }
+    attribute {
+      name: "else_branch"
+      g {
+        node {
+          input: "result_24"
+          output: "result_28"
+          name: "n0"
+          op_type: "Identity"
+          domain: ""
+        }
+        name: "elseGraph_27"
+        output {
+          name: "result_28"
+          type {
+          }
+        }
+      }
+      type: GRAPH
+    }
+    domain: ""
+  }
+  opset_import {
+    domain: ""
+    version: 18
+  }
+  domain: "pkg.onnxscript.torch_lib"
+}
+functions {
+  name: "Rank"
+  input: "input"
+  output: "return_val"
+  node {
+    input: "input"
+    output: "tmp"
+    name: "n0"
+    op_type: "Shape"
+    domain: ""
+  }
+  node {
+    input: "tmp"
+    output: "return_val"
+    name: "n1"
+    op_type: "Size"
+    domain: ""
+  }
+  doc_string: "Take the rank of the input tensor."
+  opset_import {
+    domain: ""
+    version: 18
+  }
+  domain: "pkg.onnxscript.torch_lib.common"
+}
+functions {
+  name: "IsScalar"
+  input: "input"
+  output: "return_val"
+  node {
+    input: "input"
+    output: "tmp"
+    name: "n0"
+    op_type: "Shape"
+    domain: ""
+  }
+  node {
+    input: "tmp"
+    output: "tmp_0"
+    name: "n1"
+    op_type: "Size"
+    domain: ""
+  }
+  node {
+    output: "tmp_1"
+    name: "n2"
+    op_type: "Constant"
+    attribute {
+      name: "value_int"
+      i: 0
+      type: INT
+    }
+    domain: ""
+  }
+  node {
+    input: "tmp_0"
+    input: "tmp_1"
+    output: "return_val"
+    name: "n3"
+    op_type: "Equal"
+    domain: ""
+  }
+  doc_string: "Return whether the input has rank 0, or is a scalar."
+  opset_import {
+    domain: ""
+    version: 18
+  }
+  domain: "pkg.onnxscript.torch_lib.common"
+}
+
+"""
+
+ort_inputs = {'input_0': array(0.8965, dtype=float16)}
+
+# Set up the inference session
+session_options = ort.SessionOptions()
+session_options.graph_optimization_level = ort.GraphOptimizationLevel.ORT_DISABLE_ALL
+onnx_model = onnx.ModelProto()
+google.protobuf.text_format.Parse(onnx_model_text, onnx_model)
+
+# Uncomment this line to save the model to a file for examination
+# onnx.save_model(onnx_model, "test_output_match_opinfo__linalg_vector_norm_cpu_float16.onnx")
```

Comment:
this model name doesn't match the newly added test model name - is it supposed to?